src/Clinic.DAL/ClinicDbContext.cs

```diff
@@ -10,16 +10,38 @@ public class ClinicDbContext : DbContext
 
         public ClinicDbContext(DbContextOptions<ClinicDbContext> options, IConfiguration configuration) : base(options)
         {
+            Database.EnsureDeleted();
```

Comment:
imagine, a db in production is deleted every time you do a request

Reply:
i understand, but without this compiler gave me exception on previous migration that table Office is already exists

Reply:
ok, it's working without this line